manifest.json

```diff
@@ -22,5 +22,16 @@
       "isRequired": true,
       "isBackendOnly": false
     }
+  },
+  "proxy": {
+    "whitelist": [
+      {
+        "url": "https://www.google.com/s2/favicons?domain=.*",
```

Comment:
Did you confirm that we're ok to use this service? i.e. does Google's licence permit redistribution usage?

Reply:
I asked David if this is fine, and he said it should be..